source/standards/understanding-risks.html.md.erb

```diff
@@ -0,0 +1,43 @@
+---
+title: Understand the risks to your service
+expires: 2019-03-01
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+When you build, maintain or change your service you must have a clear understanding of any associated threats and risks because they will impact your service design.
+
+You should work with the [GDS Information Assurance (IA)][] team to understand your service’s risks and design appropriate solutions. In some scenarios IA may need to obtain risk acceptance from your Senior Information Risk Owner (SIRO) before you can go ahead with certain activities.
+
+The service manual has some recommendations about how you can:
+
+* [protect against fraud][] when you design and manage your service
+* [secure your information][] if the data you handle is classed as ‘official’
+
+## Model security threats
+
+You can gain a clearer understanding of the threats to your service by [modelling them][].
+
+GDS typically does threat modelling by running [Attack Tree][] development workshops. Any workshops you run should involve a mixture of disciplines to help cover all vectors of attack.
+
+If you’re unsure about how to run an Attack Tree workshop the Cyber Security team can provide guidance, attend workshops, or help facilitate them.
+
+You can contact Cyber Security using the [#security Slack channel][] or by email using [security-team@digital.cabinet-office.gov.uk][]
+
+## Further Reading
```

Comment:
I added this section, do you think there's anything else we could share with people?